pandas/core/strings/accessor.py

```diff
@@ -3640,7 +3640,7 @@ def casefold(self):
     >>> s3 = pd.Series(['23', '³', '⅕', ''])
     >>> s3.str.isdigit()
     0     True
-    1     True
+    1    False
```

Comment:
This is a behaviour change we should actually fix, see https://github.com/pandas-dev/pandas/issues/61466